### DISCUSSION TITLE
Group wise Page Break

### DISCUSSION THREAD

**Discussion by VermaYograj** [QuestPDF/QuestPDF#174] [Q&A] [answered]
Can we get the number of records in one page, so that we can break the page accordingly.

**Comment by MarcinZiabek** [comment, marked as answer]
Hello 😁 I have just released the QuestPDF 2022.5 version. It introduces the `DynamicComponent` structure that gives more precise control over the content visible on each page. [Please take a look if it solves your requirement.](https://www.questpdf.com/documentation/patterns-and-practices.html#dynamic-components)

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
